### ISSUE TITLE
Addon for Thunderbird 52.*?

### ISSUE THREAD

**Issue by xianwenchen** [richq/smileyfixer#11]
Hi!

I guess the addon works for any version of Thunderbird, including the latest 52.*? On the official website, there is a restriction of Thunderbird versions 3.0.1 - 38.*: 
https://addons.mozilla.org/en-US/thunderbird/addon/smiley-fixer/

Cheers,
Xianwen

**Comment by richq** [comment]
Yes, it works with 52.3.0, I use it every day with the latest stable TB.

Does the add-ons site not let you install it? I thought they just ignored it there now, otherwise I'd have to make no-change releases just to bump the supported version number.

**Comment by xianwenchen** [comment]
Hello Richard,

Thank you!

Exactly. When searching the add-on within Thunderbird, no result is 
returned (because of version issue). It will be great if you could make 
a no-change release! Is there a way to just remove the version limit in 
the add-on?

Cheers,

Xianwen

**Comment by ArthurHoaro** [comment]
Thunderbird won't let me install it directly from the website, but if click on « Download anyway », and then install it directly from the file, it works just fine.

![image](https://user-images.githubusercontent.com/1962678/33551500-d39db692-d8f1-11e7-9387-4312ad7d1453.png)

**Comment by ChristofKaufmann** [comment]
It installs flawlessly on TB 60.*. Thank for fixing. Can be closed now.
